libs/permissions/permission-storage-reader/src/main/kotlin/net/corda/libs/permissions/storage/reader/PermissionStorageReader.kt

```diff
@@ -30,6 +30,14 @@ interface PermissionStorageReader : Resource {
      */
     fun publishUpdatedUser(user: AvroUser)
 
+    /**
+     * Broadcasts the deleted user onto the messaging bus.
+     *
+     * @param loginName The login name of the user to be deleted.
+     * @param user The user to be published.
+     */
+    fun publishDeletedUser(loginName: String, user: AvroUser?)
```

Comment:
I wonder if we ever going to pass a non-null `AvroUser` to this method?
If not, it would probably make more sense:
```suggestion
    fun publishDeletedUser(loginName: String)
```